.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,23 +4,63 @@ about: Create a report to help us improve
 labels: 'bug'
 ---
 
+**Bug was detected in one of the following services:**
+<!-- Please choose in which service was the bug. -->
+
+[ ] - tedge-agent
+
+[ ] - tedge-mapper-collectd
+
+[ ] - tedge-mapper-c8y
+
+[ ] - tedge-mapper-az
+
+[ ] - tedge-mapper-aws
+
+[ ] - tedge-agent
```

Comment:
As @rina23q, I like @reubenmiller 's suggestion to use labels and to tag issues appropriately when the bug is discovered or better understood.

Reply:
I feel it's better to have `logs` pre-fixed with the `tedge component name`. Then it will help us to understand which component is failing. Then no need of selecting the component.